components/camel-nats/src/main/java/org/apache/camel/component/nats/NatsConfiguration.java

```diff
@@ -454,4 +460,55 @@ public boolean isTraceConnection() {
     public void setTraceConnection(boolean traceConnection) {
         this.traceConnection = traceConnection;
     }
+
+    /**
+     * Whether to enable JetStream support for this endpoint.
+     */
+    public boolean isJetstreamEnabled() {
+        return jetstreamEnabled;
+    }
+
+    /**
+     * Sets whether to enable JetStream support for this endpoint.
+     *
+     * @param jetstreamEnabled {@code true} to enable JetStream, {@code false} otherwise.
```

Comment:
Please remove the param, is not needed. We are going to generate documentation from the javadoc, this would mess the documentation page.

Reply:
OK.